.gitignore

```diff
@@ -99,3 +99,4 @@ Tools/ssl/amd64
 Tools/ssl/win32
 .vs/
 .vscode/
+.idea/
```

Comment:
Note that you can use a global .gitignore file to ignore these IDE specific files. We've rejected a few PRs because the global .gitignore solution is the preferred way to do it.